src/commons/utils.jsx

```diff
@@ -27,3 +27,20 @@ export function participantToString(participant, t, accounts) {
 	}
 	return participant?.fullName || participant?.name || participant?.address || '';
 }
+
```

Comment:
just a suggestion ,you can directly create a hook for this instead of function (although it's just a suggestion)